packages/logs/src/domain/logsCollection/console/consoleCollection.ts

```diff
@@ -19,24 +19,25 @@ const LogStatusForApi = {
   [ConsoleApiName.warn]: StatusType.warn,
   [ConsoleApiName.error]: StatusType.error,
 }
-export function startConsoleCollection(configuration: LogsConfiguration, sender: Sender) {
+export function startConsoleCollection(configuration: LogsConfiguration, lifeCycle: LifeCycle) {
   const consoleObservable = initConsoleObservable(configuration.forwardConsoleLogs)
   const consoleSubscription = consoleObservable.subscribe(reportConsoleLog)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏`consoleObservable` could be inlined